docs/bias_correction.rst

```diff
@@ -1,22 +1,84 @@
+
+`Callable workflow`_ : ``apply_annual_blendmorph``, ``apply_interval_blendmorph``
 
 Spatially Consistent Bias Correction: Conditioned (SCBC_C)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Conditioned Spatially Consistent Bias Correction (SCBC_C) combines the meterlogic conditioning elements of `IBC_C <Independent Bias Correction: Conditioned (IBC_C)>`_ with the spatial consistency of `SCBC_U <Spatially Consistent Bias Correction: Univariate (SCBC_U)>`_. This implementation of SCBC factors in meterologic variables given into the formulation of refernce flows for each seg to be corrected to. Defined by the hydrologic response units, or <hru's <data.rst/Common Naming Conventions>`_, they impact, meterologic data is mappable to each seg within the watershed topology. In `IBC_C <Independent Bias Correction: Conditioned (IBC_C)>`_, only the data mapped to gauge sites would be used in bias correction, whereas SCBC_C can utilize meterologic data across the watershed as it incoporates all defined segs. 
+
+`Callable workflow`_ : ``apply_annual_blendmorph``, ``apply_interval_blendmorph``
+
+.. _`Callable workflow`: https://bmorph.readthedocs.io/en/develop/api.html#module-bmorph.core.workflows
+
+bmorph Parameters
+-----------------
+
+Conditioning: edcdfm vs mdcdedcdfm
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+EDCDFm
+""""""
+
+Before describing how conditioning is integrated into ``bmorph`` bias correction, we need to discuss the standard method. 
+``bmorph`` implements part of PresRat bias correction from Pierce et al. (2015), which is an extension of Equidistant quantile matching (EDCDFm) technique of Li et al. (2010). ``bmorph`` uses the amended EDCDFm to compute multiplicative changes in the quantiles of a Cumulative Distribution Fuction (CDF). Here, only flow values are used to construct the CDFs. 
+
+.. image:: Figures/Quantile_Map_Diagram.png
+    :alt: Quantile mapping is shown between a Modeled CDF curve (purple) and Observed CDF curve (grey), where x_modeled is inputed to the moodeled curve to get q_modeled, which is compared to Observed at q_modeled to get x_corrected.
+
+mDCDEDCDFm
+""""""""""
+
+multiDimensional ConDitional EquiDistant CDF matching function, (mDCDEDCDFm), incorporates meteorlogic data into the ``bmorph`` bias correction process to condition flow time series to other hydrologically relevant information. By creating a series of CDFs based on meteorlogic data, (such as minimum daily temperature), ``bmorph`` can select a CDF that will not only correct the time series, but most closely match the metrologic conditions simulated.
+    
+.. image:: Figures/conditioning_diagram_with_arrows.png
```

Comment:
These arrows look a bit fuzzy, can you remake so they are sharper?

Reply:
Should be fixed, uploaded the image you created leading up to AGU